### PR TITLE
Development branch line 50

### PR DESCRIPTION
**Pull request by fdacunha22**
Review the development branch

### CODE REVIEW

**Comment by fdacunha22** [comment]
Check the new code

**Review by FiodorGrecu — COMMENTED** [review]
For all the duplicates I suggest that we delete them. Fernando, please assign someone from the team to do that.

**Comment by bashiraPoya** [comment]
Can you please assign someone to format the code? It is very messy.

**Comment by bashiraPoya** [comment]
Do we really need line 19? I think we should get ride of that.

File: src/team/g4/DemoG4.java [hunk 1]
```diff
@@ -46,6 +46,8 @@ public static void thatsEasy() {
 
 	public static void lastName(String st) {
 		System.out.println("Poya");
+		
+		//Checking the GitHub Development branch
```
Comment (bashiraPoya) [inline src/team/g4/DemoG4.java:50]: I like the changes in here.

**Review by bashiraPoya — COMMENTED** [review]
Good!
[fahim's notes.txt](https://github.com/NTKBatch5/G4/files/9264353/fahim.s.notes.txt)

**Pull request merged**